src/functions-and-arrays.js

```diff
@@ -1,41 +1,76 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
-
+function maxOfTwoNumbers(a, b) {
+  if (a > b) {
+    return a;
+  } else if (b > a) {
+    return b;
+  } else {
+    return a || b;
+  }
+}
+maxOfTwoNumbers(5, 4);
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
-
-
+function findLongestWord(words) {
+  if (words.length === 0) {
+    return null;
+  } else {
+    return words.reduce((a, b) => (a.length < b.length ? b : a), '');
+  }
+}
```

Comment:
Good that you used reduce but take a look on the lab solution to also be familiar with other ways to do it.